fs/accounting/stats.go

```diff
@@ -66,7 +68,8 @@ func (s *StatsInfo) RemoteStats() (out rc.Params, err error) {
 	out["transfers"] = s.transfers
 	out["deletes"] = s.deletes
 	out["renames"] = s.renames
-	out["elapsedTime"] = s.totalDuration().Seconds()
+	out["transferTime"] = s.totalDuration().Seconds()
+	out["elapsedTime"] = time.Now().Sub(startTime).Seconds()
```

Comment:
This is usually written `time.Since(startTime)` instead of `time.Now().Sub(startTime)` - it means the same thing though!

Reply:
That's pretty neat.